packages/devtools_app/lib/src/code_size/code_size_table.dart

```diff
@@ -125,3 +125,69 @@ class _SizePercentageColumn extends ColumnData<TreemapNode> {
   @override
   double get fixedWidthPx => 100.0;
 }
+
+class CodeSizeDiffTable extends StatelessWidget {
+  factory CodeSizeDiffTable({@required rootNode}) {
+    final treeColumn = _NameColumn(currentRootLevel: rootNode.level);
+    final columns = List<ColumnData<TreemapNode>>.unmodifiable([
+      treeColumn,
+      _DiffColumn(),
```

Comment:
can we include the diff bytes and percent columns as well? like here: https://pub.dev/packages/vm_snapshot_analysis#compare. you may need to look into the vm_snapshot_analysis package code to see how percent was calculated

Reply:
> How does this look in light mode?

* Light mode
<img width="1552" alt="Screen Shot 2020-07-01 at 3 39 04 PM" src="https://user-images.githubusercontent.com/22206026/86284590-22763880-bbb1-11ea-8acb-8eecc4c7f862.png">